1-exercises/G-array-methods/exercise.js

```diff
@@ -4,7 +4,7 @@
 */
 
 let numbers = [3, 2, 1];
-let sortedNumbers; // complete this statement
+let sortedNumbers = numbers.sort(); // complete this statement
```

Comment:
when using .sort() I would create a copy of the array, so that you know there is no mutation happening. for example:
`let sortedNumbers = [...numbers].sort();`
This way you know that  your original numbers array is not going to be mutated, you can read more about the spread operator here https://www.javascripttutorial.net/es6/javascript-spread/